lib/data/content/chapter-8.yml

```diff
@@ -46,9 +48,11 @@ case_studies:
+    
+    Oceans are complex systems, and marine connectivity can be estimated and predicted differently depending on the approaches applied and metrics used (Bryan-Brown <em>et al.</em>, 2017). Ocean circulation, habitat structures, nutrients and genetics can exhibit diverse connectivity patterns. A multidisciplinary approach is therefore fundamental to obtaining the best guidelines for incorporating connectivity into Marine Protected Area (MPA) network design (Balbar and Metaxas, 2019). \n
+          Although a standardised global methodology for assessing marine connectivity is yet to be developed, there has been extensive research conducted to evaluate connectivity across the Wider Caribbean region (Truelove <em>et al.</em>, 2015; Chollett <em>et al.</em>, 2017; Knowles <em>et al.</em>, 2015; Schill <em>et al.</em>, 2015). A recent pilot study under the <a title='Visit BIOPAMA' href='https://www.biopama.org/' target='_blank'>BIOPAMA programme</a> adapted the ProtConn indicator to assess the importance of protected areas for seagrass habitat connectivity across the Caribbean<sup>2</sup>. This study identified several data challenges and considerations for the marine realm, which will be incorporated into the development of a global marine ProtConn indicator<sup>1</sup>. The Figure above shows the configuration of seagrass across the Caribbean, 20% of which is under protection from sites recorded in the WDPA. When comparing the functional connectivity of the seagrass habitat network, regardless of protection status, to that of the network containing only protected sites, the unprotected areas boost the overall available habitat while increasing its relative reachability, largely through increases in stepping-stone connectivity.\n
+          <sup>1</sup> ProtConn: Protected Connected (ProtConn) is an indicator developed by The European Commission’s Joint Research Centre that quantifies the percentage of a region covered by protected connected lands, and differentiates several categories of land (unprotected, protected or transboundary) through which movement between protected locations may occur (European Commission, 2017).\n
+          <sup>2</sup> Data provided by <a href='https://caribbeanscienceatlas.tnc.org/' title='Visit The Nature Conservancy' target='_blank'>The Nature Conservancy</a>\n
+          This analysis allows the identification of protected areas whose connection to the network is particularly vulnerable and where extra buffering or linking could be considered. The method provides additional benefits by prioritising areas where strategic interventions are feasible, highlighting the importance of comprehensive habitat maps that, when paired with local knowledge, can inform design and management decisions."
```

Comment:
Should this be after the footnotes like this? If so, fine 
<img width="945" alt="image" src="https://user-images.githubusercontent.com/56026099/118280968-9f0dbf00-b4c4-11eb-84a9-9602c43e8d55.png">